src/shared/inc/gcinfo.h

```diff
@@ -67,15 +82,11 @@ struct GCInfoToken
 
     static uint32_t ReadyToRunVersionToGcInfoVersion(uint32_t readyToRunMajorVersion, uint32_t readyToRunMinorVersion)
     {
-        // Once MINIMUM_READYTORUN_MAJOR_VERSION is bumped to 10+
-        // delete the following and just return GCINFO_VERSION
-        //
-        // R2R 9.0 and 9.1 use GCInfo v2
-        // R2R 9.2 uses GCInfo v3
-        if (readyToRunMajorVersion == 9 && readyToRunMinorVersion < 2)
-            return 2;
-
+#ifdef SOS_INCLUDE
+        return GCInfoVersion();
```

Comment:
As commented above, we can just use v4 or v3 here. 
There is no difference for scenarios in SOS between v2 and v3 - they are exactly the same file format.